### DISCUSSION TITLE
corner_radius doen't work ?? noo it's cavnas transparency :(

### DISCUSSION THREAD

**Discussion by MaxNiark** [TomSchimansky/CustomTkinter#1333] [Ideas]
CTkFrame is child of CTKCavnas, (child of tk.tkinter) (Cf.Module CL_CTkCavnas, CL_CTkFrame) 

As Tk.tkinter don't integrate "transparency" on him Canvas(tk.tkinter)

the only way to see correctly the corner_radius is to set the bg_color as the same than back_ground of display Widget(Canvas,Frame,ScrolableFrame, Etc......) 

 ![image](https://user-images.githubusercontent.com/126392328/224497134-01f62b03-3807-46b6-993e-b3fcb2389636.png)

**Comment by Akascape** [comment]
@MaxNiark A new solution for this issue: https://github.com/TomSchimansky/CustomTkinter/discussions/2214
